src/test/java/be/syntra/devshop/DevshopBack/testutilities/CartUtils.java

```diff
@@ -1,70 +1,81 @@
 package be.syntra.devshop.DevshopBack.testutilities;
 
-import be.syntra.devshop.DevshopBack.entities.Cart;
+import be.syntra.devshop.DevshopBack.entities.OrderContent;
 import be.syntra.devshop.DevshopBack.entities.Product;
+import be.syntra.devshop.DevshopBack.entities.ShopOrder;
 import be.syntra.devshop.DevshopBack.models.CartDto;
+import be.syntra.devshop.DevshopBack.models.CartProductDto;
 
 import java.time.LocalDateTime;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.stream.Collectors;
 
 import static be.syntra.devshop.DevshopBack.testutilities.ProductUtils.createDummyNonArchivedProductList;
+import static be.syntra.devshop.DevshopBack.testutilities.ProductUtils.createProductDto;
 
 public class CartUtils {
 
 
-    public static Cart createCart() {
+    public static ShopOrder createCart() {
         List<Product> products = createDummyNonArchivedProductList();
-        return Cart.builder()
-                .products(products)
-                .cartCreationDateTime(LocalDateTime.now())
-                .finalizedCart(false)
-                .paidCart(false)
+        return ShopOrder.builder()
+                .orderContents(
+                        products.stream()
+                                .map(product -> OrderContent.builder()
```

Comment:
OrderContent.builder() appears twice. Is eligible for it's own method.

Reply:
extracted